src/Eloquent/Model.php

```diff
@@ -191,13 +196,14 @@ public function setAttribute($key, $value)
             $value = $builder->convertKey($value);
         } // Support keys in dot notation.
         elseif (Str::contains($key, '.')) {
```

Comment:
Well, if we do remove dot notation support in dates, we need to remove this here as well.

Reply:
Dot notation support wasn't removed, see below.